app/vue3/src/client/preview/render.ts

```diff
@@ -0,0 +1,58 @@
+import type { Args } from '@storybook/addons';
+import dedent from 'ts-dedent';
+import { createApp, h, shallowRef, ComponentPublicInstance } from 'vue';
+import { RenderContext, StoryFnVueReturnType } from './types';
+
+const activeStoryComponent = shallowRef<StoryFnVueReturnType | null>(null);
+const activeProps = shallowRef<Args>({});
```

Comment:
Is a shallow ref fine here? I don't think these props need to be wrapped in a `reactive`, right?